renovate.json

```diff
@@ -4,7 +4,7 @@
   "extends": ["config:base"],
   "prConcurrentLimit": 10,
   "branchConcurrentLimit": 0,
-  "prHourlyLimit": 5,
+  "prHourlyLimit": 10,
```

Comment:
I would update the schedule as well to get PRs more frequently. Looking at the dashboard, it looks like there are some pending updates.

Reply:
I think schedule is fine, to avoid noise from dependency updates. We'll do this once a week, not more often.
I'd say let's try this, then we can see if backlog clears fast enough.